### PR TITLE
added class people and created p1 in main

### PR DESCRIPTION
**Pull request by PiumaP**
please check my part of code

### CODE REVIEW

File: src/com/company/People.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.company;
+
+public class People {
+    String name;
```
Comment (hvma411) [inline src/com/company/People.java:4]: You should use private here :)

File: src/com/company/People.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    @Override
+    public String toString() {
```
Comment (hvma411) [inline src/com/company/People.java:49]: make this function more readable

**Review by hvma411 — CHANGES_REQUESTED** [review]

File: src/com/company/People.java (hunk 1 above)
Comment (PiumaP) [reply src/com/company/People.java]: done

File: src/com/company/People.java (hunk 2 above)
Comment (PiumaP) [reply src/com/company/People.java:49]: done

**Pull request merged**